### PR TITLE
#167484610 implemented article reading statistics 

### PR DESCRIPTION
**Pull request by placideirandora**
#### What does this PR do?
It implements the approach of being able to retrieve articles' reading statistics.

#### Description of Task to be done?
1. Adding code to record a specific article's retrieval.
2. Adding a route for retrieving an author's articles' reading statistics.
3. Adding a controller for retrieving an authors' articles' reading statistics.
4. Adding tests for reading statistics.

#### Screenshots
Screen Shot 2019-09-02 at 21 46 26

Screen Shot 2019-09-02 at 21 47 18

#### How should this be manually tested?
1. Clone this branch.
2. Do **npm install**.
4. Do **npm start**
5. Create an article.
6. Check how many times it has been read with the following endpoint: **/api/v1/articles/{articleSlug}/statistics**
7. Retrieve the article that you have created  with the following endpoint: **/api/v1/articles/{articleSlug}**
8. Re-Check how many times it has been read with the following endpoint: **/api/v1/articles/{articleSlug}/statistics** and you will find that it has been read once. Therefore, the more you retrieve it, the more reads increase**

### CODE REVIEW

File: src/helpers/articles/recordStats.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Statistic } from '../../sequelize/models';
+
+const recordStatistics = async (article) => {
```
Comment (malaba6) [inline src/helpers/articles/recordStats.js:3]: This line 3 is not fully tested 
Screen Shot 2019-08-30 at 16 59 08

Comment (placideirandora) [reply src/helpers/articles/recordStats.js:3]: I don't know why it didn't get fully covered but it really does its job of receiving an article as a parameter.

**Review by harerakalex — APPROVED** [review]
Looks good

**Review by Jkadhuwa — APPROVED** [review]
Looks fine and works as intended.

**Pull request merged**